src/content/smart-bulletins/030.md

```diff
@@ -0,0 +1,88 @@
+
+### OMB Circular A-123, Appendix B Chapter 5.3.1 – Statistical Reporting –
+Requested information in the template attached below shall be submitted, at a minimum, on an annual fiscal year basis. Agency-provided information shall be furnished to CCCM *no later than 90 calendar days after the end of the Fiscal Year (FY)*. The due date for agency/organization submission of FY2019 agency statistical reporting to CCCM is hereby extended to No Later Than (NLT) February 28, 2020. This extension applies only to the FY 19 reports.
+
+The Statistical Reporting requirement applies to **all business lines** under the GSA SmartPay program. Agencies must report the following items for each business line:
+
+- Count of accounts, as defined by GSA;
+- Total number of accounts across the charge card program as compared to the prior reporting period;
+- Total charge card dollars spent; and
+- Total refunds earned.
+
+Specific requirements for each business line are as follows:
+
+*Purchase cards only (to be provided to CCCM by CFO Act Agencies)* 
+- Ratio of A/Os to purchase cardholders (span of control);
+- Number of purchase cards with transaction limits over the micro-purchase threshold
+- Ratio of the number of confirmed violations reported pursuant to P.L. 112-194 as compared to the number of valid transactions within the reporting period (GSA's Center for Charge Card Management will obtain confirmed violation information from OMB/OFFM).
+
+*Convenience checks only:*
+- Total number of convenience checks written;
+- Number of checks written over micro-purchase threshold;
+- Number of checks written under micro-purchase threshold; and
+- Total dollar amount of checks written.
+
+*Fleet cards only (to be provided to CCCM by CFO Act Agencies):*
+- Ratio of reviewing officials to fleet cards (span of control); and
+- Number of fleet cards with transaction limits over micro-purchase threshold. Integrated cards only (to be provided to CCCM by CFO Act Agencies):
+- Ratio of A/Os to integrated cardholders (purchase business line - span of control);
+- Ratio of reviewing officials to integrated cards (fleet business line - span of control);
+- Number of integrated cards with transaction limits over the micro-purchase threshold and
+- Ratio of the number of confirmed violations reported pursuant to P.L. 112-194 as compared to the number of valid transactions within the reporting period (GSA's Center for Charge Card Management will obtain confirmed violation information from OMB/OFFM).
+
+### OMB Circular A-123, Appendix B Chapter 5.3.2 – Narrative Reporting – 
+For agencies/organizations subject to the CFO Act, the following information should be updated annually on a fiscal year basis and provided to CCCM *no later than 90 calendar days after the end of each fiscal year:*
```

Comment:
For agencies/organizations subject to the CFO Act, the following information should be updated annually on a fiscal year basis and provided to CCCM no later than 90 calendar days after the end of each fiscal year:
The date(s) of most recent and next scheduled independent review(s) (e.g., OIG audits and risk assessments) for all agency charge card programs;
A description of the current process for monitoring delinquency, including what reports the agency reviews and what actions are taken when a problem is discovered;
A description of any best practices the agency employs in charge card management;
Progress reports on agency integration of GSA SmartPay e-payable payment process into their accounts payable processes and
Any additional information regarding agency management and use of charge card programs, including innovative applications to streamline business processes, increased revenue generation from expanded use for payments, and creative internal controls that maintain program integrity, flexibility, and ease of use.
The due date for submission of the FY2019 agency narrative reporting to CCCM is similarly extended to NLT on February 28, 2020. This extension applies only to FY 19 reports. For FY2020 reports and beyond, agency narrative reports are due to CCCM NLT 90 calendar days after the end of the fiscal year requirement as specified in the current version of OMB Circular A-123, Appendix B.